### PR TITLE
README Improvement

### PR DESCRIPTION
**Pull request by takuti**
Add screenshots and update README contents. You can preview this: https://github.com/Sushisushi-sandesu/sushi/blob/readme-improvement/README.md

Plz review guys :bow:  (especially @natezzz )

### CODE REVIEW

**Comment by takuti** [comment]
なぜかDecemberコミットで余計なスペース入ってしまっていて精神衛生上よろしくないけど12月なので大目に見てください :bow:

**Comment by tomoasleep** [comment]
(For reviewers,) you can see the content of the new README [here](https://github.com/Sushisushi-sandesu/sushi/blob/readme-improvement/README.md).

File: README.md [hunk 1]
```diff
@@ -1,59 +1,66 @@
-[![Stories in Ready](https://badge.waffle.io/Sushisushi-sandesu/sushi.png?label=ready&title=Ready)](https://waffle.io/Sushisushi-sandesu/sushi)
-# Sushi
+# Sushi :sushi: [![Stories in Ready](https://badge.waffle.io/Sushisushi-sandesu/sushi.png?label=ready&title=Ready)](https://waffle.io/Sushisushi-sandesu/sushi)
 
-##How to use the code
+:snowman: :star2: :tada: :christmas_tree: :christmas_tree: :christmas_tree: :tada: :star2: :snowman:
```
Comment (ganmacs) [inline README.md:3]: :sunglasses:

**Comment by takuti** [comment]
@natezzz plz review~~~~

**Comment by natezzz** [comment]
@takuti ok

**Comment by takuti** [comment]
@natezzz ok to merge?

**Comment by natezzz** [comment]
@takuti Everything is correct and beautifully organized.
Ok to merge.

**Comment by takuti** [comment]
@natezzz thanks!

**Pull request merged**